src/cljs/athens/views/pages/node_page.cljs

```diff
@@ -430,66 +430,51 @@
-                       :width "100%"}}
-         [:span unlinked?]
-         (when (and unlinked? (not-empty @unlinked-refs))
-           [button {:style    {:font-size "14px"}
-                    :on-click (fn []
-                                (let [unlinked-str-ids (->> @unlinked-refs
-                                                            (mapcat second)
-                                                            (map #(select-keys % [:block/string :block/uid])))] ; to remove the unnecessary data before dispatching the event
-                                  (dispatch [:unlinked-references/link-all unlinked-str-ids title]))
-
-                                (swap! state assoc unlinked? false)
-
-                                (reset! unlinked-refs []))}
-            "Link All"])]]
-       (when (get @state unlinked?)
-         [:div (use-style references-list-style)
-          (doall
-            (for [[group-title group] @unlinked-refs]
-              [:div (use-style references-group-style {:key (str "group-" group-title)})
-               [:h4 (use-style references-group-title-style)
-                [:a {:on-click #(navigate-uid (:block/uid @(pull-node-from-string group-title)) %)} group-title]]
-               (doall
-                 (for [block group]
-                   ^{:key (str "ref-" (:block/uid block))}
-                   [:div {:style {:display         "flex"
-                                  :justify-content "space-between"
-                                  :align-items     "flex-start"}}
-                    [:div (merge
-                            (use-style references-group-block-style)
-                            {:style {:max-width "90%"}})
-                     [ref-comp block]]
-                    (when unlinked?
-                      [button {:style    {:margin-top "1.5em"}
-                               :on-click (fn []
-                                           (let [hm                (into (hash-map) @unlinked-refs)
-                                                 new-unlinked-refs (->> (update-in hm [group-title] #(filter (fn [{:keys [block/uid]}]
-                                                                                                               (= uid (:block/uid block)))
-                                                                                                             %))
-                                                                        seq)]
-                                             ;; ctrl-z doesn't work though, because Unlinked Refs aren't reactive to datascript.
-                                             (reset! unlinked-refs new-unlinked-refs)
-                                             (dispatch [:unlinked-references/link block title])))}
-                       "Link"])]))]))])])))
+  [_ title]
+  (let [unlinked-refs (r/track get-unlinked-references (escape-str title))]
+    (fn [state _]
+      (let [is-unlinked-ref? (:is-unlinked-ref? @state)]
```

Comment:
Clojure convention is to skip `is-` part, `?` makes it a predicate already.

Reply:
Updated
https://media3.giphy.com/media/cdNSp4L5vCU7aQrYnV/giphy.gif?cid=36b14fac1o34mrqauoe0alswpj5s8o3ku84ml5q6nhck7j9q&rid=giphy.gif&ct=g

Reply:
@juniusfree I think Alex was asking for you to change it the other way around: from `is-unlinked-ref` to `unlinked-ref?`,